src/manager/params.rs

```diff
@@ -0,0 +1,14 @@
+use cid::Cid;
+use fvm_ipld_encoding::RawBytes;
+use fvm_shared::MethodNum;
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+
+/// Init actor Exec Params, see https://github.com/filecoin-project/builtin-actors/blob/master/actors/init/src/types.rs#L17
+#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
+pub(crate) struct ExecParams {
+    pub code_cid: Cid,
+    pub constructor_params: RawBytes,
+}
+
+/// Init actor exec method number, see https://github.com/filecoin-project/builtin-actors/blob/fb759f87fcd5de0a98cb61966cd27f680df83364/actors/init/src/lib.rs#L32
+pub(crate) const INIT_EXEC_METHOD_NUM: MethodNum = 2;
```

Comment:
Let's make this available in the `fvm_utils` instead of here. We are keeping all of these builtin_actors and fvm-specific things in that repo. This crate is a good candidate: https://github.com/consensus-shipyard/fvm-utils/tree/main/runtime/src/builtin

You could also pick it up from the `builtin_actors` but that may lead to additional overheads to comply with their version updates.

Reply:
right right, this parameter is not used, I forgot to delete.

Reply:
Then also `ExecParams` params too?

Reply:
+1